triage/2024-07-30.md

```diff
@@ -0,0 +1,289 @@
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | -     | -              | 0     |
+| Regressions ❌ <br /> (secondary)  | 2.1%  | [2.1%, 2.1%]   | 1     |
+| Improvements ✅ <br /> (primary)   | -0.2% | [-0.3%, -0.2%] | 2     |
+| Improvements ✅ <br /> (secondary) | -1.6% | [-3.0%, -0.2%] | 2     |
+| All ❌✅ (primary)                 | -0.2% | [-0.3%, -0.2%] | 2     |
+
+* sole regression is to secondary benchmark coercions debug-full.
+* seems like noise.
+* marked as triaged
+
+ Perform instsimplify before inline to eliminate some trivial calls  [#128265](https://github.com/rust-lang/rust/pull/128265) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=56c698c71130de6fe55ba703a161405b6145b90e&end=4db3d12e6f395babed53dee1d209a5c8699a5ae6&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 1.2%  | [0.2%, 2.6%]   | 4     |
+| Regressions ❌ <br /> (secondary)  | 0.5%  | [0.5%, 0.5%]   | 1     |
+| Improvements ✅ <br /> (primary)   | -0.5% | [-0.8%, -0.2%] | 12    |
+| Improvements ✅ <br /> (secondary) | -0.3% | [-0.4%, -0.3%] | 2     |
+| All ❌✅ (primary)                 | -0.0% | [-0.8%, 2.6%]  | 16    |
+
+* main primary regressions are to ripgrep opt full and image opt-full
+* these changes were anticipated during review, seems likely result of changes to inlining decisions
+* marked as triaged
+
+Rollup of 6 pull requests [#128360](https://github.com/rust-lang/rust/pull/128360) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=612a33f20b9b2c27380edbc4b26a01433ed114bc&end=368e2fd458a22d0cc133d0c254f2612ee999744f&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.6%  | [0.4%, 0.7%]   | 4     |
+| Regressions ❌ <br /> (secondary)  | 4.4%  | [0.3%, 12.0%]  | 10    |
+| Improvements ✅ <br /> (primary)   | -0.3% | [-0.3%, -0.3%] | 4     |
+| Improvements ✅ <br /> (secondary) | -     | -              | 0     |
+| All ❌✅ (primary)                 | 0.1%  | [-0.3%, 0.7%]  | 8     |
+
+
+* primary regressions are to doc-full for html5ever, stm32f4, libc, and typenum
+* those are presumably due to PR [#126247](https://github.com/rust-lang/rust/issues/126247); pnkfelix thinks the above not worth further investigation
+* however, Kobzol has pointed out that the secondary regressions are significant, and has identified the root cause as PR [#128104](https://github.com/rust-lang/rust/issues/128104)
+* we are in any case planning to revert the changes to dead code analysis (see PR [#128404](https://github.com/rust-lang/rust/issues/128404)) which should address those regressions.
+* marked as triaged.
+
+#### Untriaged Pull Requests
```

Comment:
it seems you did triage a bunch if not all all of the "untriaged pull requests" below?

Reply:
uh, maybe you meant to say "didn't triage"? I definitely didn't look at that list at all; I'm late enough delivering this report as it is...